packages/std/src/testing/mock.rs

```diff
@@ -89,6 +103,20 @@ pub fn mock_dependencies_with_balances(
     }
 }
 
+/// Initializes the querier along with the mock_dependencies.
+/// Sets all balances provided (you must explicitly set contract balance if desired).
+pub fn mock_dependencies_with_balances_valid(
```

Comment:
This name is more of a placeholder because I couldn't come up with a good one.
Any ideas?